views/default/user.html

```diff
@@ -16,375 +16,342 @@
 
 <div class="container rscontainer">
 
-{{if request.args(0) != 'register':}}
```

Comment:
Is this just all stuff that was reformatted by some html tidy script or are there real and substantial changes in here?  This is the "front door" to runestone so huge changes like this with no context make me nervous?

Reply:
I did a lot of manual reformatting, then made some fixes.

IMHO, the reformatting makes the HTML usable -- the nesting is so broken, it's very difficult to make any changes. Code should be usable and modifiable. Github is for history. The HTML formatting currently is just horrible.

Plus...hidden in all that formatting mess is broken HTML. Which is what millions of users are experiencing currently. Why not serve correct, validated HTML instead?

I understand nervousness. But, this PR demonstrates that EVERY file now validates properly, which is a big deal. More testing leads to less nervousness. If this breaks something, then we should be testing that something.